packages/@o3r/extractors/src/core/wrapper.ts

```diff
@@ -20,17 +20,16 @@ export const createBuilderWithMetricsIfInstalled: BuilderWrapper = (builderFn) =
   let wrapper: BuilderWrapper = noopBuilderWrapper;
   try {
     const { createBuilderWithMetrics } = await import('@o3r/telemetry');
-    if (packageJson.config?.o3rMetrics) {
-      wrapper = createBuilderWithMetrics;
-    }
+    wrapper = createBuilderWithMetrics;
```

Comment:
This is not exactly correct: in `npm` context, this dep can be accessible because part of a dep dependency but you may have disable the metrics in your project.
It is also enabling telemetry per default if no `o3rMetrics` is specified :S.
I think the previous was actually more correct in that case

Reply:
The wrapper handle all the logic to know if data should be sent or not 
with the environment variable etc...